### PR TITLE
Updates to settings.yaml for ntrip username/password/gga_out_interval

### PR DESCRIPTION
**Pull request by benjaminaltieri**
Updates to settings per the following buildroot PRs: 
* swift-nav/piksi_buildroot#525
 * swift-nav/piksi_buildroot#527

### CODE REVIEW

File: piksi_tools/console/settings.yaml [hunk 1]
```diff
@@ -1391,6 +1391,28 @@
...
+- group: ntrip
+  name: password
+  expert: false
+  type: string
+  units: N/A
+  default value: N/A
+  readonly: false
+  enumerated possible values:
+  Description: NTRIP password to use. The interface will receive 1002, 1004, 1005, 1006, 1010, 1012, 1033 and 1230 RTCMv3.1 messages and will not transmit or receive any other messages.
+  Notes: Password to use with NTRIP client. NTRIP must be enabled to use this setting.
```
Comment (cbeighley) [inline piksi_tools/console/settings.yaml]: Can you give some guidance as to what special characters are / are not allowed here?

File: piksi_tools/console/settings.yaml [hunk 2]
```diff
@@ -1391,6 +1391,28 @@
...
+- group: ntrip
+  name: username
+  expert: false
+  type: string
+  units: N/A
+  default value: N/A
+  readonly: false
+  enumerated possible values:
+  Description: NTRIP username to use. The interface will receive 1002, 1004, 1005, 1006, 1010, 1012, 1033 and 1230 RTCMv3.1 messages and will not transmit or receive any other messages.
+  Notes: Username to use with NTRIP client. NTRIP must be enabled to use this setting.
```
Comment (cbeighley) [inline piksi_tools/console/settings.yaml]: Can you give some guidance as to what special characters are / are not allowed here?

File: piksi_tools/console/settings.yaml (hunk 1 above)
Comment (benjaminaltieri) [reply piksi_tools/console/settings.yaml]: I couldn't find any characters the broke it. So no guidance may be better than imprecise guidance.

**Comment by denniszollo** [comment]
Doesn't this PR require a master equivalent? @benjaminaltieri ?

**Comment by denniszollo** [comment]
superseded by #732

**Pull request closed without merging**